cli/src/main/scala/io.buildo.metarpheus/cli/Cli.scala

```diff
@@ -30,14 +32,16 @@ object Cli {
     
     val sources = files.map(Source.fromFile(_).mkString)
 
-    val config = conf.configPath.get.map { fileName =>
-      val eval = new com.twitter.util.Eval(None)
-      eval.apply(new File(fileName)): core.Config
-    }.getOrElse(core.DefaultConfig)
+    implicit val parseConfig: Configuration = Configuration.default.withDefaults
+    val config = (for {
+      fileName <- conf.configPath.get
+      json = Source.fromFile(fileName).mkString
+      parsed <- decode[core.Config](json).toOption
+    } yield parsed).getOrElse(core.Config.default)
 
     val wiro = conf.wiro.get.getOrElse(false)
 
-    val api = core.Metarpheus.run(sources, config, wiro)
+    val api = core.Metarpheus.run(sources, config.copy(wiro = wiro))
```

Comment:
I intend to remove `wiro` from the flags and have it only as a config key, so this is a temporary workaround.